src/lib.rs

```diff
@@ -92,13 +85,13 @@ pub type AppId = Vec<u8>;
 #[derive(Debug)]
 pub enum RegisterResult {
     CTAP1(Vec<u8>, u2ftypes::U2FDeviceInfo),
-    CTAP2(AttestationObject, CollectedClientDataWrapper),
+    CTAP2(AttestationObject),
```

Comment:
Do we still need `RegisterResult` (and `SignResult`) with the `CTAP1` and `CTAP2`-variants? Isn't it now always `CTAP2`, aka `AttestationObject` here and `AssertionObject` for signing?

Reply:
The `CTAP1` variant is still used in `virtualdevices/software_u2f.rs`. I'm going to update that soon, and I will simplify `RegisterResult` and `SignResult` at the same time.